ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/AppCard.tsx

```diff
@@ -37,58 +37,69 @@ export function AppCard({ app, className, envSlug }: Props) {
   return (
     <div
       className={classNames(
-        'flex w-full min-w-[800px] max-w-[1200px] overflow-hidden rounded-lg border border-slate-300 bg-white',
+        'flex h-56 w-full min-w-[800px] max-w-[1200px] overflow-hidden rounded-lg border border-slate-300 bg-white',
         className
       )}
     >
-      <div className="m-4 mt-8 flex w-64 items-center border-r border-slate-400">
+      <div className="bg-slate-910 flex w-[410px] flex-col justify-center gap-2 px-10">
         <h2>
           <Link
-            className="text-indigo-600 hover:underline"
+            className="transition-color flex cursor-pointer items-center gap-1 text-white underline decoration-transparent decoration-2 underline-offset-4 duration-300 hover:text-indigo-400 hover:decoration-indigo-400"
             href={`/env/${envSlug}/apps/${encodeURIComponent(app.externalID)}`}
           >
             {app.name}
+            <ChevronRightIcon className="h-4 w-4" />
           </Link>
         </h2>
+        {app.latestSync?.url && (
+          <dl>
+            <dt className="hidden">URL</dt>
+            <dd className="text-slate-400">{app.latestSync?.url}</dd>
+          </dl>
+        )}
       </div>
+      <div className="flex flex-1 items-center px-8">
+        <dl className="grid grow grid-cols-2 gap-4 md:grid-cols-3">
+          {/* Row 1 */}
+          <Description
+            className="col-span-2"
+            detail={
+              app.latestSync && (
+                <div className="flex gap-2">
+                  <SyncStatus status={app.latestSync?.status} />
+                  <Link
+                    className="transition-color flex cursor-pointer items-center gap-1 text-indigo-400 underline decoration-transparent decoration-2 underline-offset-4 duration-300  hover:decoration-indigo-400"
+                    href={`/env/${envSlug}/apps/${encodeURIComponent(app.externalID)}/syncs`}
+                  >
+                    <Time value={app.latestSync?.createdAt} />
+                    <ChevronRightIcon className="h-4 w-4" />
+                  </Link>
+                </div>
+              )
+            }
+            term="App ID"
+          />
+          <Description detail={app.latestSync?.sdkVersion} term="SDK Version" />
 
-      <dl className="m-4 grid grow grid-cols-3 gap-4">
-        {/* Row 1 */}
-        <Description detail={app.externalID} term="App ID" />
-        <Description
-          className="col-span-2"
-          detail={
-            app.latestSync && (
-              <div className="flex gap-2">
-                <Time value={app.latestSync?.createdAt} />
-                <SyncStatus status={app.latestSync?.status} />
-              </div>
-            )
-          }
-          term="App ID"
-        />
-
-        {/* Row 2 */}
-        <Description detail={app.latestSync?.sdkVersion} term="SDK Version" />
-        <Description
+          {/* Row 2 */}
+          {/* <Description
```

Comment:
Can we delete this instead of commenting out?

Reply:
I had that in a patch for the next PR, but I'm just gonna push that and one of the empty states to this one already